### ISSUE TITLE
Add Gabbard plots

### ISSUE THREAD

**Issue by astrojuanlu** [poliastro/poliastro#1431]
See for example

![savetweetvid_FHA8kccUcAIyIix](https://user-images.githubusercontent.com/316517/146810204-48ce7192-3c20-41ed-b373-e37d90951fe9.gif)

(we can do the static ones first 😉 )

**Comment by Ln11211** [comment]
Hello,
I was just wondering if this issue is open for contribution and if it is beginner-friendly? If yes, Can you please provide more details about this?

**Comment by astrojuanlu** [comment]
Hi @Ln11211 ! This issue is open for contribution. However, we usually use "beginner-friendly" for issues that are a 1 line change, or trivially explained in the issue itself, and they're just for people to get used to git and contributing 😃

This is a bit more involved, but essentially I imagine something like

```python
from poliastro.plotting.gabbard import GabbardPlotter

gd = GabbardPlotter()
gd.plot_orbits([orbit for orbit in orbits])
```

and

```python
import matplotlib.pyplot as plt

fig, ax = plt.subplots()

gd = GabbardPlotter(ax=ax)
```

And a static plot similar to the one in the first comment should show.

**Comment by Ln11211** [comment]
@astrojuanlu, so I understand that we want to produce gabbard plots for the satellite debris (If I am not wrong, it was that same satellite which was destroyed in a Russian missile test which later posed a risk to ISS crew right ? 🤔). How are the debris orbit altitude and time fed as an input for the graphs ?, so, my task is to input altitude and time and produce more such plots on different dates right?

**Comment by astrojuanlu** [comment]
The orbital data would be passed as a list of `poliastro.twobody.orbit.Orbit` objects in this case, so we would only need to write the code to do the plotting.

**Comment by Ln11211** [comment]
ok, I am assuming this will be similar to `porkchop.py` and I am really interested in doing this but I don't know for certain if I can do this(Idk how to do it, I lack the knowledge, I looked at all other files in `plotting` but I feel confused). If it's ok with you, can I write a mail to you?

**Comment by astrojuanlu** [comment]
Yeah, the plotting code is quite nested (I'm somewhat happy with the design, but I don't claim it's easy to understand initially). On the other hand, `porkchop.py` is much more amenable.

You are welcome to send me an email with general advice applied to your specific circumstances but I'd rather have discussion in public if it's applicable to the project :) This issue tracker or [our chat](http://chat.poliastro.space) are good places.

**Comment by Ln11211** [comment]
No, I don't have any advice, I want a piece of advice from you. 

I have seen your readme(your profile), and it says you contributed to the Mars helicopter(cool badge 😎), I also know that you were Libre Space mentor for a project and I believe that's not what most can achieve, so you seem like the right person to ask for advice.
Firstly,  how to start with scientific contributions? 

Secondly, What technologies (I know for certain that Python is important and what else ?) and prerequisites (like math or knowledge of data structures, etc) are necessary to make meaningful contributions?

Thirdly, it is common these days to choose a tech stack when we develop something, what tech stacks do you consider when trying to contribute to such scientific projects?

I apologize as this is in no way has a relation to this issue, that is the reason why I wanted to write a mail.

**Comment by astrojuanlu** [comment]
How do you start: you're at the right place :D
What technologies: Python is the language of science and will be for a long time! Also R some statistical analysis. Just have a look at what gets published in the Journal of Open Source Software: https://joss.theoj.org/
Prerequisites: if you want to contribute to the scientific/algorithmic part of it, then a you'll need to have a bit of familiarity with the domain. Basic Calculus, Algebra and Statistics will be the baseline, but then every field is different (in our case, Astrodynamics, we make heavy use of nonlinear algebraic equation solving aka zero finding).

But scientific packages need a lot of non-science as well: good docs, good infra (continuous integration, quality checks), a nice website, good social media presence, talks at conferences, an influx of new contributors, user support. There's dozens of ways to contribute to a scientific project that don't involve "the science".

Tech stacks: Python + GitHub + pytest + Sphinx will go a long way :)

**Comment by Ln11211** [comment]
Ok,
thank you for the reply.

**Comment by TheBuffer** [comment]
> The orbital data would be passed as a list of `poliastro.twobody.orbit.Orbit` objects in this case, so we would only need to write the code to do the plotting.

Hello, I'm interested in contributing. I'm a little confused with the data part. I'm guessing one would need external satellite fragmentation orbital data to maybe experiment around with? If so, could you perhaps point me towards this data?

**Comment by astrojuanlu** [comment]
Hi @TheBuffer! This functionality should work with a bunch of `Orbit` objects. If you want real-world data to experiment, have a look at https://docs.poliastro.space/en/latest/examples/Loading%20OMM%20and%20TLE%20satellite%20data.html

**Comment by astrojuanlu** [comment]
This was implemented in #1464! 🎉 

![gabbard](https://user-images.githubusercontent.com/316517/153395030-07fbb712-acf0-4d4d-a7fb-f1526d88d5d0.png)
